recipes/plate-simulation/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "plate-simulation" %}
+{% set version = "0.1.1" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MiraGeoscience/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ceb6c139b7c2d855822ab33348e93d0af0f7ef4bfce163145ffda130950c9aba
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core >=1.0.0
+    - setuptools
+    - pip
+  run:
+    - python >={{ python_min }},<3.11
+    # Mira packages
+    - geoapps-utils >=0.4.0,<0.5.dev
+    - geoh5py >=0.10.1,<0.11.dev
+    - octree-creation-app >=0.2.0,<0.3.dev
+    - param-sweeps >=0.2.0,<0.3.dev
+    - simpeg-drivers >=0.2.1,<0.3.dev
+    # other direct dependencies
+    - numpy >=1.26.0,<1.27.dev
+    - pydantic >=2.5.2,<2.6.dev
+    - rtree >=1.2.0,<1.3.dev
+    - scipy >=1.14.0,<1.15.dev
+    - threadpoolctl >=3.3.0,<3.4.dev
+    - trimesh >=4.1.3,<4.2.dev
+  run_constrained:
+    - __glibc >=2.17
```

Comment:
Are you sure you need this? This may create issues with environment solves.

Reply:
good question. It comes from the recipe of `simpeg-driver` (see: https://github.com/conda-forge/simpeg-drivers-feedstock/blob/main/recipe/recipe.yaml )
We needed it there to ensure we could run it on older systems.
But maybe this constraint is not needed here, since already in the `simpeg-drivers` dependency?
I am not exactly sure how this works. What do you recommend?

Reply:
Let's remove it and see if the build passes.